frontend/schema/class-schema-webpage.php

```diff
@@ -78,13 +82,40 @@ public function generate() {
 
 		if ( $this->add_breadcrumbs() ) {
 			$data['breadcrumb'] = array(
-				'@id' => $this->context->canonical . WPSEO_Schema_IDs::BREADCRUMB_HASH,
+				'@type' => 'Person',
+				'@id'   => $this->context->canonical . WPSEO_Schema_IDs::BREADCRUMB_HASH,
 			);
 		}
 
 		return $data;
 	}
 
+	/**
+	 * Determine if we should add an author attribute.
```

Comment:
This description sounds like it will return a boolean, which it doesn't.

Reply:
Changed